ui-tests/src/test/java/org/openmrs/reference/TransferToWardServiceTest.java

```diff
@@ -30,23 +32,21 @@
     @Before
     public void createTestData() throws Exception {
         testPatient = createTestPatient();
-        String visitType = TestData.getAVisitType();
-        String visit = new TestData.TestVisit(testPatient.uuid, visitType, getLocationUuid(homePage)).create();
```

Comment:
Refactored the `codeblock ` above so that the test logic is effective. Ideally `transferToWardService` test implementation can be effective when interacting with `PatientVisitDashboard` page **not** `ClinicianFacingPatientDashboard page`. The refactored `codeblock` creates `testvisit.uuid, visistType` and creating `SessionLocation `(`getLocationUuid(homePage`), But creating session location can only be effective when a test class extends `LocationSensitiveApplicationTestBase` so this block `getLocationUuid(homePage)` is not used by the test at all . The other `two methods'` implementation has been catered within the test logic method `LN 40-43` and thus enabling the test to interact with `PatientVisitDashboard` page which holds functionalities that the test implementation needs for it to be effective.